Tensile/Common.py

```diff
@@ -1720,68 +1737,76 @@ def locateExe( defaultPath, exeName ): # /opt/rocm/bin, hip-clang
 
 def GetAsmCaps(isaVersion):
   """ Determine assembler capabilities by testing short instructions sequences """
-  rv = {}
-  rv["SupportedISA"]          = tryAssembler(isaVersion, "")
-  rv["HasExplicitCO"]         = tryAssembler(isaVersion, "v_add_co_u32 v0,vcc,v0,1")
-  rv["HasExplicitNC"]         = tryAssembler(isaVersion, "v_add_nc_u32 v0,v0,1")
+  if globalParameters["AssemblerPath"] != None:
```

Comment:
```suggestion
  if globalParameters["AssemblerPath"] is not None:
```
It is recommended to use is/is not for None checking